middleware_static_auth.go

```diff
@@ -0,0 +1,112 @@
+package gimlet
+
+import (
+	"net/http"
+	"path"
+	"strings"
+)
+
+// StaticAuth is a middleware handler that serves static files in the given
```

Comment:
Can we comment that this is copied directly from negroni's `StaticAuth`, except the last bit that involves our own authentication system?